stan/math/opencl/matrix_cl.hpp

```diff
@@ -9,7 +9,7 @@
 #include <stan/math/opencl/err/check_opencl.hpp>
 #include <stan/math/prim/arr/fun/vec_concat.hpp>
 #include <stan/math/prim/scal/err/check_size_match.hpp>
-#include <stan/math/prim/scal/err/domain_error.hpp>
```

Comment:
This import can be removed (`domain_error` was never used in this code).

Reply:
Done.